### PR TITLE
add multicluster-gateways profile

### PR DESCRIPTION
**Pull request by linsun**

### CODE REVIEW

File: content/docs/setup/kubernetes/additional-setup/config-profiles/index.md [hunk 1]
```diff
@@ -29,29 +29,32 @@ your specific needs. The following built-in configuration profiles are currently
 1. **remote**: used for configuring remote clusters of a
     multicluster mesh with a [single control plane topology](/docs/concepts/multicluster-deployments/#single-control-plane-topology).
 
+1. **multicluster-gateways**: used for configuring multicluster gateways of
+    multicluster mesh with [multiple control plane topology](/docs/concepts/multicluster-deployments/#multiple-control-plane-topology).
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/additional-setup/config-profiles/index.md]: a multicluster

**Comment by linsun** [comment]
i signed it

**Comment by linsun** [comment]
validated the table looks okay.  PTAL

**Comment by sdake** [comment]
@linsun the core components are not correct.  See screenshot:
Screen Shot 2019-03-15 at 10 14 42 PM

Following docs here:
https://preliminary.istio.io/docs/setup/kubernetes/install/multicluster/gateways/

Yields the following components from the profile:
```
sdake@beast-01:~/go/src/istio.io/istio$ kubectl get pods -n istio-system
NAME                                                         READY   STATUS              RESTARTS   AGE
istio-citadel-675f6688fb-lmmt4                               0/1     Pending             0          5s
istio-cleanup-secrets-release-1.1-latest-daily-jvtr8         0/1     ContainerCreating   0          6s
istio-egressgateway-6747f494b9-lhblf                         0/1     ContainerCreating   0          5s
istio-galley-68f6c89b64-c862j                                0/1     ContainerCreating   0          5s
istio-ingressgateway-5ffcc5bfbb-bh47k                        0/1     ContainerCreating   0          5s
istio-pilot-5bbf96d6b7-wxzwz                                 0/2     Pending             0          5s
istio-policy-577f44d8c6-pzjmt                                0/2     Pending             0          5s
istio-security-post-install-release-1.1-latest-daily-vqjcm   0/1     ContainerCreating   0          6s
istio-sidecar-injector-5bf4d7c598-blfx9                      0/1     Pending             0          5s
istio-telemetry-7779787f8-f4qg2                              0/2     Pending             0          5s
istiocoredns-bd48b885d-47555                                 0/2     Pending             0          5s
prometheus-66c9f5694-kzzf9                                   0/1     Pending             0          5s
```

**Comment by linsun** [comment]
@sdake thanks, I miss read the helm value file, i'll correct it shortly

**Comment by linsun** [comment]
@sdake is it possible your cluster is old level of istio?  I don't understand why mixer policy is enabled in yours.

**Comment by linsun** [comment]
@sdake PTAL.  (ignore the mixer policy issue, I raised it on slack)

**Comment by linsun** [comment]
@frankbu SG thanks!

**Pull request closed without merging**